modules/drivers/druid/src/metabase/driver/druid.clj

```diff
@@ -13,8 +13,9 @@
 
 (driver/register! :druid)
 
-(doseq [[feature supported?] {:set-timezone            true
-                              :expression-aggregations true}]
+(doseq [[feature supported?] {:expression-aggregations true
+                              :schemas                 false
+                              :set-timezone            true}]
```

Comment:
What's the significance of this change? As small as it is, I dislike it being smuggled inside this refactor if it's unrelated.

Reply:
I explained that [here](https://github.com/metabase/metabase/pull/41588#discussion_r1578360819).

Reply:
What UI are you using to leave these comments? I'm surprised you missed it, because I see my comment directly above your comment.
<img width="656" alt="image" src="https://github.com/metabase/metabase/assets/39073188/af70b714-150a-47a3-a08b-933bdc09381c">

Reply:
I'm talking about the timezone.

Reply:
Unfortunate the UI isn't clearer on exactly which lines were commented on.